addons.yaml

```diff
@@ -169,12 +169,13 @@ microk8s-addons:
         - amd64
         - arm64
 
-    - name: "gopaddle-lite"
-      description: "Cheapest, fastest and simplest way to modernize your applications"
-      version: "4.2.5"
+    - name: "gopaddle"
+      description: "DevSecOps and Multi-Cloud Kubernetes Platform"
+      version: "latest"
```

Comment:
Please install a specific version. Latest is not good because the version you install here will be in the 1.28 release and should continue deploy and work as you move forward with gopaddle.

Reply:
@Gayathri-Bluemeric any update on this?

Reply:
@ktsakalozos After talking with the gopaddle team, it appears that the best option is to use the `latest` version of gopaddle as an add-on. Right now, the gopaddle add-on is a few releases behind due to its dependency on the version. We noticed that there are other add-ons leveraging the `latest` version, so we want to try the same. Whenever a new version of microk8s comes out, we will test gopaddle with that version. Furthermore, whenever gopaddle releases a newer version, we will qualify it with the most recent version of microk8s. Additionally, we will create a support matrix to indicate which versions of gopaddle are backward compatible with the older versions of microk8s.

Let me know your thoughts on this.

Reply:
It was a mistake on our part to allow addons deploying the `latest` version. We are trying to address this mistake.

When we release microk8s on any track we run the integration tests ensuring the addons deploy correctly. If an addon does not pass the tests it blocks the release. We also want to release the same day as upstream. As you understand we cannot test the latest release of all the addons as it would be almost certain that some addon will be misbehaving at the time of a release. It is reasonable to distribute a specific version of each addon with a specific version of each K8s. This should not come as a surprise, I am sure for example that each gopaddle release is build against specific versions of library you depend on. 

> whenever gopaddle releases a newer version, we will qualify it with the most recent version of microk8s

It would not be enough to test gopaddle with the latest MicroK8s release. Consider this case. In a year from now the latest MicroK8s would be v1.31, however the v1.28, 1.29, 1.30 tracks would still be shipping new releases. Either way gopaddle should be testing its releases with all supported K8s releases.

Another reason we cannot ship the `latest` release of each addon is that the UX would be inconsistent. Imagine the surprise of a user deploying everyday a specific version of MicroK8s (eg in his CI) and finding that features have been removed.

It would be best if the addon would deploy a version of gopaddle that is within a release track, ie it would be updating only to include bug fixes ensuring backwards compatibility.

Reply:
Hello @ktsakalozos, I have opened a new pull request with the version changes. I have removed the `latest` version and added the current version of Gopaddle.

Reply:
@ktsakalozos  This makes a lot of sense. We got some feedback from the community earlier on why we attached a specific version to the add-on and the microk8s users were trying the older version of gopaddle even though we were 5 releases ahead. But I agree with you that adding a version instead of  the `latest`, would be more predictable. I have reverted the changes. Would you be able to review the new PR ?